docs/storing-querying/storage-options/index.md

```diff
@@ -60,15 +60,17 @@ All lake loaders are micro-batching.
 
 | Lake | Format | Compatibility | Loader application | Status |
 | --- | --- | --- | --- | --- |
+| S3 | Delta    | Amazon Athena | [Lake Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/index.md) | Early release |
```

Comment:
Should compatibility here include Databricks?

Reply:
Yes you're right, that's my mistake.

Did you notice I did not list Hudi in this table?  I figured that since we've never yet connected it to a warehouse, we can't say for certain that it's compatible with anything.